gen/AArch64Arch_gen.ml

```diff
@@ -597,6 +597,26 @@ let get_ie e = match e with
 
 let fold_edge f r = Code.fold_ie (fun ie r -> f (IFF ie) (f (FIF ie) r)) r
 
+
+let compute_rmw r old co =
+   begin match r with
+    | LdOp op ->
+      begin match op with
+        | A_ADD -> old + co
+        | A_SMAX -> if old > co then old else co
+        | A_SMIN -> if old < co then old else co
+        | A_EOR | A_SET | A_CLR -> co
```

Comment:
I know this is wip, but if in the end we don't add support for bitwise ops yet, maybe we could add a warning.